apps/daimo-mobile/src/view/shared/InfoToast.tsx

```diff
@@ -0,0 +1,205 @@
+import { SuggestedAction } from "@daimo/api";
+import Octicons from "@expo/vector-icons/Octicons";
+import { useEffect, useState } from "react";
+import { Linking, StyleSheet, View } from "react-native";
+import { PanGestureHandler } from "react-native-gesture-handler";
+import Animated, {
+  runOnJS,
+  useAnimatedGestureHandler,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+  withTiming,
+} from "react-native-reanimated";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+
+import { OctName } from "./InputBig";
+import { handleDeepLink, useNav } from "./nav";
+import { color } from "./style";
+import { TextBody } from "./text";
+import { useAccount } from "../../model/account";
+
+const ADDITIONAL_TOP_PADDING = 4;
+
+export function InfoToast({ action }: { action: SuggestedAction }) {
+  const nav = useNav();
+  const [account, setAccount] = useAccount();
+  const { icon, title, subtitle } = action;
+  const [isVisible, setIsVisible] = useState(true);
+  const ins = useSafeAreaInsets();
+  const top = Math.max(ins.top, 16);
+  const y = useSharedValue(-100);
+  const opacity = useSharedValue(0);
+  const scale = useSharedValue(1);
+
+  const wasCancelled = useSharedValue(false);
+
+  const onPress = () => {
+    setIsVisible(false);
+
+    if (action.url.startsWith("http")) {
+      Linking.openURL(action.url);
+    } else if (action.url.startsWith("daimo")) {
+      handleDeepLink(nav, action.url);
+    }
+
+    if (account) {
+      setAccount({
+        ...account,
+        suggestedActions:
+          account?.suggestedActions?.filter(
+            (a: SuggestedAction) => a.id === action.id
+          ) || [],
+      });
+    }
```

Comment:
can be deduped with below

not sure if this is necessary btw... just filtering from `suggestedActions` won't do much, since `suggestedActions` is overwritten on each sync

all good--i'll experiment with what feels right for completed actions